clamav/clamav.lua

```
@@ -1,318 +1,297 @@
-	return self:ret(true, "no file detected")
+-- Enhanced multipart parsing function with proper HTTP/2 support
+function clamav:parse_multipart(body, boundary)
+  if not body or not boundary then
+    return {}
+  end
+  
+  local parts = {}
+  
+  -- Split body by boundary markers
+  local sections = {}
+  local current_pos = 1
```

Luacheck (1.2.0)

[warning] value assigned to variable 'current_pos' is unused

(W311)